cli/src/commands/shell/index.ts

```diff
@@ -154,13 +154,17 @@ function getDefaultEval() {
   return defaultEval;
 }
 
-function postProcess(result: any, options: { table: boolean }, callback: (err: Error | null, result: any) => void) {
+function postProcess(
+  result: any,
+  options: { table: boolean; deep?: boolean },
+  callback: (err: Error | null, result: any) => void
+) {
   const { table } = options;
   return callback(
     null,
     table
       ? console.table(result)
-      : typeof result === 'object'
+      : typeof result === 'object' && options.deep
```

Comment:
Why are you adding the `deep` flag? Don't we want that not only in raw API calls, but in JS statements as well?

Reply:
Tudor noticed that autocomplete is also using postprocessing and had some weird behavior. I'll forward you the video.

Added it as an opt-in flag, and let's decide where to add it.